src/test/java/com/epam/tc/hw2/ex1/Ex1.java

```diff
@@ -0,0 +1,174 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+
+
+
+public class Ex1 {
+
+
+    WebDriver driver;
+
+    @BeforeSuite
+    static void setupAll() {
+        WebDriverManager.chromedriver().setup();
+
+    }
+
+    @BeforeTest
+    void setup() {
+        driver = new ChromeDriver();
+    }
+
+
+    @Test
+    //1. Open test site by URL
+    public void ex1() {
+
```

Comment:
Methods should be verbs, in mixed case with
the first letter lowercase, with the first letter of
each internal word capitalized.

from https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
Fixed a bit. Code divided into separate logical classes and methods in the following work.